etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,73 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+
+	advClientUrls := "http://" + hostname + ":2380"
+
+	etcdStart := exec.Command("/bin/etcd",
+		"-listen-client-urls", listenClientUrls,
+		"-advertise-client-urls", listenClientUrls,
+		"-listen-peer-urls", advClientUrls,
+		"-initial-advertise-peer-urls", advClientUrls,
+		"--initial-cluster", "default="+advClientUrls)
+
```

Comment:
I had asked you to use different variable names for all these parameters, it doesn't harm us if the values are duplicate since the readability will improve.